### ISSUE TITLE
[Web][Accessibility][11yMAS][MAS2.4.3][Focus is moving to hidden elements after Yammer link]

### ISSUE THREAD

**Issue by DasaradhaKamjula** [OfficeDev/script-lab-2017#675]
**User Experience:**
If focus moves to hidden elements keyboard user will not be able to identify where the focus is present.

**Test Environment:**
OS Version: 16299.19
Browser: Microsoft Edge 41.16299.15.0
**Attachment:**
[MAS2.4.3_Focus Moves to hidden elements.zip](https://github.com/OfficeDev/script-lab/files/1440321/MAS2.4.3_Focus.Moves.to.hidden.elements.zip)
[MAS2.4.3_Hidden elements.zip](https://github.com/OfficeDev/script-lab/files/1445112/MAS2.4.3_Hidden.elements.zip)
[MAS2.4.3 Hidden Controls_MAC.zip](https://github.com/OfficeDev/script-lab/files/1456650/MAS2.4.3.Hidden.Controls_MAC.zip)

**Repro Steps:**
1.Open the URL:
[http://scriptlabdocsdemo.azurewebsites.net/Office%20Dev%20Center%20-%20Docs%20-%20Build%20your%20first%20Excel%20add-in.html](url)
2.Navigate to Yammer link and press Tab key.
3.Verify focus is moving to hidden elements after Yammer link or not..

**Current Behavior:**
Focus is moving to hidden elements after Yammer link.

**Expected Behavior:**
Focus should not go to hidden elements which will make user to search for Tab focus.

**Note:**
1. Same issue Repro after Documentation Menu item, Focus is moving to hidden element.
2. Same issue is repro for the [MAC + safari][ Build info: OS Build: Mac OS Sierra 0.12.6 Browser: Safari 11.0.1] Focus navigates to hidden elements on the page.

**Observation:**
Focus is moving to hidden element and announcing it as "Addthis sharing buttons email link" .In this there is no space between **"add"** and **"this"** words , so screen reader will think it as single word and read same which will confuse user.
**Attachment:** 
![space should be maintained between two words](https://user-images.githubusercontent.com/33054846/32534962-f588df08-c47d-11e7-9227-105c1b912f90.png)

**MAS Reference:**
**MAS 2.4.3:**
[https://microsoft.sharepoint.com/:w:/r/teams/msenable/_layouts/15/WopiFrame.aspx?sourcedoc=%7B0de7fbe1-ad7e-48e5-bcbb-8d986691e2b9%7D](url)

**Suggested Fix:**
[https://microsoft.sharepoint.com/teams/msenable/mas/pages/mas-details.aspx?masid=2.4.3](url)
